packages/versions/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fuel-ts/versions",
-  "version": "0.94.9",
+  "version": "0.94.10",
```

Comment:
Before merging this one, let's double-check if we don't want to add anything else.

Perhaps it is good to wait a bit more, just in case.